.travis.yml

```diff
@@ -10,10 +10,10 @@ before_install:
   - echo "MAVEN_OPTS='-Dmaven.repo.local=$HOME/.m2/repository -Xmx1g -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss:SSS'" > ~/.mavenrc
 
 install:
-  - ./mvnw dependency:resolve -B
+  - ./mvnw dependency:resolve -B || true
 
 script:
-  - ./mvnw checkstyle:check -B
+  - ./mvnw -DskipTests package checkstyle:check -B
```

Comment:
I had to add this too. Maven doesn’t like doing checkstyle before compiling.

Reply:
Maven _(shakes fist)_